modules/ROOT/pages/clauses/load-csv.adoc

```diff
@@ -397,4 +568,383 @@ If `LOAD CSV` is invoked with a `file:///` URL that points to your disk `file()`
 +------------------------------------------+
 1 row
 ----
+====
+
+[TIP]
+`file()` always returns a local path, even when loading remote CSV files.
+For remote resources, `file()` returns the temporary local path it was downloaded to.
+
+
+== CSV file format
+
+The CSV file format and `LOAD CSV` interact as follows:
+
+* the file character encoding must be UTF-8;
+* the line terminator is system dependent (`\n` for Unix and `\r\n` for Windows);
+* the default field delimiter is `,`. Change it with the option `FIELDTERMINATOR`;
+* CSV files may contain quoted string values, and the quotes are dropped when `LOAD CSV` reads the data;
+* if `dbms.import.csv.legacy_quote_escaping` is set to the default value of `true`, `\` is used as an escape character;
+* a double quote must be in a quoted string and escaped, with either the escape character or a second double quote.
```

Comment:
Bullet point lists should be capitalized and each point should end with a full stop.